<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowFoMPlot</class>
 <widget class="QMainWindow" name="WindowFoMPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>567</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>UpArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Figure of Merit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="CalibratedPlot" name="qwtPlotFoM" native="true">
      <property name="canvasBackground" stdset="0">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>Configure</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionActiveCuts"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionSaveImage"/>
    <addaction name="actionSaveData"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRename_window"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfigure"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="shortcut">
    <string>C, O</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>Save image</string>
   </property>
   <property name="shortcut">
    <string>P, I</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="text">
    <string>Save data</string>
   </property>
   <property name="shortcut">
    <string>P, D</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>C, R</string>
   </property>
  </action>
  <action name="actionActiveCuts">
   <property name="text">
    <string>Active cuts</string>
   </property>
   <property name="shortcut">
    <string>C, A</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>C, C</string>
   </property>
  </action>
  <action name="actionRename_window">
   <property name="text">
    <string>Rename window</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CalibratedPlot</class>
   <extends>QWidget</extends>
   <header>CalibratedPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>WindowFoMPlot</receiver>
   <slot>onCloseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveData</sender>
   <signal>triggered()</signal>
   <receiver>WindowFoMPlot</receiver>
   <slot>onSaveDataClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveImage</sender>
   <signal>triggered()</signal>
   <receiver>WindowFoMPlot</receiver>
   <slot>onSaveImageClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>WindowFoMPlot</receiver>
   <slot>onRefreshClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOptions</sender>
   <signal>triggered()</signal>
   <receiver>WindowFoMPlot</receiver>
   <slot>onOptionsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionActiveCuts</sender>
   <signal>triggered()</signal>
   <receiver>WindowFoMPlot</receiver>
   <slot>onActiveCutsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>WindowFoMPlot</receiver>
   <slot>clearValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRename_window</sender>
   <signal>triggered()</signal>
   <receiver>WindowFoMPlot</receiver>
   <slot>onRenameClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCloseClicked()</slot>
  <slot>onSaveDataClicked()</slot>
  <slot>onSaveImageClicked()</slot>
  <slot>onRefreshClicked()</slot>
  <slot>onOptionsClicked()</slot>
  <slot>onActiveCutsClicked()</slot>
  <slot>clearValues()</slot>
  <slot>onRenameClicked()</slot>
 </slots>
</ui>
